Formula/volta.rb

```diff
@@ -0,0 +1,27 @@
+class Volta < Formula
+  desc "Volta: The JavaScript Launcher"
```

Comment:
```suggestion
  desc "The JavaScript Launcher"
```

but ideally it'd describe what it does, I have no idea from the description or homepage.

Reply:
We're going to iterate on that a bit on the website (this was helpful!) but something like "A JavaScript toolchain manager for reproducible development environments".

Reply:
Used @chriskrycho suggestion, although I had to adapt to fit Brew's requirement. So the description I went with was "The JavaScript toolchain manager for reproducible environments". Resolved on d142d754e.